examples/layer-browser/src/map.js

```diff
@@ -0,0 +1,169 @@
+import React, {PureComponent} from 'react';
+import {StaticMap, _MapContext as MapContext, NavigationControl} from 'react-map-gl';
+import autobind from 'react-autobind';
+
+import DeckGL from '@deck.gl/react';
+import {COORDINATE_SYSTEM, View} from '@deck.gl/core';
+
+import LayerInfo from './components/layer-info';
+
+/* eslint-disable no-process-env */
+const MapboxAccessToken =
+  process.env.MapboxAccessToken || // eslint-disable-line
+  'Set MapboxAccessToken environment variable or put your token here.';
+
+const NAVIGATION_CONTROL_STYLES = {
+  margin: 10,
+  position: 'absolute',
+  zIndex: 1
+};
+
+const VIEW_LABEL_STYLES = {
+  zIndex: 10,
+  // position: 'relative',
+  padding: 5,
+  margin: 20,
+  fontSize: 12,
+  backgroundColor: '#282727',
+  color: '#FFFFFF'
+};
+
+const ViewportLabel = props => (
+  <div style={{position: 'absolute'}}>
+    <div style={{...VIEW_LABEL_STYLES, display: ''}}>{props.children}</div>
+  </div>
+);
+
+export default class Map extends PureComponent {
+  constructor(props) {
+    super(props);
+    autobind(this);
+
+    this.state = {
+      mapViewState: {
+        latitude: 37.752,
+        longitude: -122.427,
+        zoom: 11.5,
+        pitch: 0,
+        bearing: 0
+      },
+      orbitViewState: {
+        target: [0, 0, 0],
+        zoom: 3,
+        rotationX: -30,
+        rotationOrbit: 30,
+        orbitAxis: 'Y'
+      },
+      hoveredItem: null,
+      clickedItem: null,
+      queriedItems: null,
+
+      enableDepthPickOnClick: false
+    };
+
+    this.deckRef = React.createRef();
+  }
+
+  pickObjects(opts) {
+    if (this.deckRef.current) {
+      const infos = this.deckRef.current.pickObjects(opts);
+      console.log(infos); // eslint-disable-line
+      this.setState({queriedItems: infos});
+    }
+  }
+
+  pickMultipleObjects(opts) {
+    if (this.deckRef.current) {
+      const infos = this.deckRef.current.pickMultipleObjects(opts);
+      console.log(infos); // eslint-disable-line
+      this.setState({queriedItems: infos});
+    }
+  }
+
+  _onViewStateChange({viewState, viewId}) {
+    if (viewId === 'infovis') {
+      this.setState({orbitViewState: viewState});
+      return;
+    }
+    if (viewState.pitch > 60) {
+      viewState.pitch = 60;
+    }
+    viewState.position = [0, 0, 50];
```

Comment:
Why z is `50`?

Reply:
Arbitrary number for testing first person view.